.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file seems to have sneaked in from an IDE configuration folder, and should be removed.